README.md

```diff
@@ -1,6 +1,125 @@
 # Sensitive Data Archive
 
-`SDA` contains all components of [NeIC Sensitive Data Archive](https://neic-sda.readthedocs.io/en/latest/) It can be used as part of a [Federated EGA](https://ega-archive.org/federated) or as a isolated Sensitive Data Archive.
+The `SDA` contains all components of [NeIC Sensitive Data Archive](https://neic-sda.readthedocs.io/en/latest/). It can be used as part of a [Federated EGA](https://ega-archive.org/federated) or as a standalone Sensitive Data Archive.
 
-For more information about the different components see the readme files in the respecive folders.
-For more information on how to start developing read [Getting Started developing components of the SDA stack](/GETTINGSTARTED.md).
\ No newline at end of file
+For more information about the different components, please refer to the README files in their respective folders.
+
+## How to run the SDA stack 
+The following instructions outline the steps to set up and run the `SDA` services for development and testing using Docker. These steps are based on the provided [Makefile](./Makefile) commands.
+
+### Prerequisites
+Ensure you have the following installed on your system:
+
+- [`Go`](https://www.golang.org/): The required version is specified in the `sda` Dockerfile. Verify using
+    ```sh
+    $ make go-version-check
```

Comment:
I got 
```
$ make go-version-check
SDA requires go 1.23 to build; found 1.22.3.
make: *** [Makefile:43: go-version-check] Error 1
```
when the version does not meet the requirement, which looks fine to me.

When it succeeds, nothing is printed at all—I suppose this is @jbygdell's style. See also the comment at
https://github.com/neicnordic/sensitive-data-archive/pull/1026#discussion_r1758350236.

Reply:
Verbosity should be handled with care.
While it is easy to add a printout of the detected version it should come as an expense of what's in the error message.

Something like this could be added after line 46:
```
echo "Installed GO version: $${GO_VERSION}."
```

And then remove `; found $${GO_VERSION}` from the error message on line 52.

Reply:
Ah, didn't see that comment in #1026. And now I also get a sensible output when having an older output (must have mistyped or somehow misbehaved when testing, sorry). I think the suggestion above looks good, but it could be kept outside of this PR if you prefer. Feel free to resolve, anyway :) .

Reply:
Yes, I also think it is better to print a clear message on pass or fail. Fixed in the commit 441cc5fe